vm/src/builtins/memory.rs

```diff
@@ -52,6 +57,8 @@ pub struct PyMemoryView {
     // instead it relay on the buffer it viewing to maintain the count
 }
 
+pub type PyMemoryViewRef = PyRef<PyMemoryView>;
```

Comment:
we are creating these shortcut type alias for only very common types. using `PyRef<PyMemoryView>` will be better.